sdk/communication/communication-network-traversal/samples-dev/getRelayConfigurationWithTtl.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Issue a new Relay configuration without user identity
+ */
+
+import { CommunicationRelayClient } from "@azure/communication-network-traversal";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set this environment variables or edit the following values
+const connectionString =
+  process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
+
+export async function main() {
+  console.log("\n== Get Relay configuration passing Time to live for credential ==\n");
+
+  const relayClient = new CommunicationRelayClient(connectionString);
+  console.log("Getting relay configuration");
+
+  const config = await relayClient.getRelayConfiguration(5000);
```

Comment:
if `ttl` was part of the options, readability would benefit as the intent of `5000` becomes clear

```typescript
const config = await relayClient.getRelayConfiguration({ttl: 5000});
```

Reply:
@joheredi Would it makes sense to add only the ttl argument as part of the options? Because we have 2 more parameters, RouteType and User identity. So, I feel like if we add one of those then we should add them all since the 3 are part of the CommunicationRelayConfigurationRequest object that we send.